ioctl/newcmd/account/accountcreate.go

```diff
@@ -57,15 +57,18 @@ func NewAccountCreate(c ioctl.Client) *cobra.Command {
 		Args:  cobra.ExactArgs(0),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
+			cmd.SilenceErrors = true
 			newAccounts := make([]generatedAccount, 0)
 			for i := 0; i < int(numAccounts); i++ {
 				private, err := crypto.GenerateKey()
 				if err != nil {
-					return output.NewError(output.CryptoError, failToGenerateNewPrivateKey, err)
+					c.PrintError(output.NewError(output.CryptoError, failToGenerateNewPrivateKey, err))
```

Comment:
can we directly use `output.PrintError(output.NewError(...))`?

Reply:
use  client.PrintError , we can test the result wether is the wanting setting by gomock .  if use output.NewError,  we cannot set the wanting by gomock, then we cannot know whether it's ok.